packages/csv-export/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.1.0
+
+- Fix null values in columns
```

Comment:
I wonder if this message could be improved a bit? _Fix JS error in `getCSVRows` when there is a null or undefined value_ or something like that?

Reply:
Ah yes, much better!  Thanks @Aljullu